controllers/datamover.go

```diff
@@ -393,6 +394,7 @@ func (r *DPAReconciler) createResticSecretsPerBSL(dpa *oadpv1alpha1.DataProtecti
 				pruneInterval = strings.ReplaceAll(pruneInterval, `"`, "")
 				pruneInterval = strings.ReplaceAll(pruneInterval, `'`, "")
 			}
+			resticCustomCA := bsl.Spec.ObjectStorage.CACert
```

Comment:
Should there be a check first as to whether or not this exists? 
`if len(bsl.Spec.ObjectStorage.CACert) > 0`

Reply:
I tried this and it didn't make sense in this particular spot because if the length is empty it just ends up passing an empty string to buildDataMoverResticSecretForAWS, so the check didn't really cause any difference in behavior.

But it does make sense to check inside of buildDataMoverResticSecretForAWS, because that way an empty CACert field doesn't get passed along to the next secret. So I added changes there instead, which I think should better cover the concern you have brought up.